waspc/cli/src/Wasp/Cli/Command/Telemetry/Project.hs

```diff
@@ -134,6 +137,7 @@ data ProjectTelemetryData = ProjectTelemetryData
     _waspVersion :: String,
     _os :: String,
     _isBuild :: Bool,
+    _deployCmd :: String,
```

Comment:
It is different though, they are different commands. Also, build is just a flag, while deploy has some stuff after it (args). And we want to differentiate them in our CLI, because we need to handle them differently.